examples/lusid/orders/blocks/0.5-call-block.sql

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
I don't think this is needed,  just confusing

Reply:
removed 0.5 call block